comp/core/flare/helpers/helpers.go

```diff
@@ -0,0 +1,179 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package helpers
+
```

Comment:
As per [this comment](https://github.com/DataDog/datadog-agent/pull/14234#discussion_r1036014999) the idea is it have the `helpers` independent from the any component logic, so it can be used by non FX-app calling `pkg/flare`.

Reply:
This recurring comment presents a valid point that should be addressed, but only _after_ all apps have been migrated to FX. I will add a card to the backlog.